Meta/lint-ci.sh

```diff
@@ -42,6 +47,16 @@ else
     ((FAILURES+=1))
 fi
 
+
+if [ "$ports" = true ]; then
```

Comment:
Short comment for our future selves about why this was split from the other lint scripts would be great :)